src/lang/english.txt

```diff
@@ -1568,6 +1568,9 @@ STR_CONFIG_SETTING_SCROLLMODE_LMB                               :Move map with L
 STR_CONFIG_SETTING_SMOOTH_SCROLLING                             :Smooth viewport scrolling: {STRING2}
 STR_CONFIG_SETTING_SMOOTH_SCROLLING_HELPTEXT                    :Control how the main view scrolls to a specific position when clicking on the smallmap or when issuing a command to scroll to a specific object on the map. If enabled, the viewport scrolls smoothly, if disabled it jumps directly to the targeted spot
 
+STR_CONFIG_SETTING_TOOLBAR_DROPDOWN_CLOSE                       :Toolbar dropdowns activate on release: {STRING2}
+STR_CONFIG_SETTING_TOOLBAR_DROPDOWN_CLOSE_HELPTEXT              :Control whether tthe main toolbar dropdowns activate their default function when released, or if they require an additional selection.
```

Comment:
This sentence is really hard to read, and even after reading it 4 times, I am not exactly sure what it does :P Maybe dumb it down a bit?